ui/src/MemeGallery.tsx

```diff
@@ -36,6 +65,8 @@ function MemeGallery(props: any) {
         <h1>Welcome to Abstract Memery</h1>
         <ul>{memes}</ul>
       </header>
+      <Button disabled={page === 0} onClick={getPreviousPage}>Previous</Button>
+      <Button disabled={lastPage} onClick={getNextPage}>Next</Button>
```

Comment:
Rather than Previous and Next buttons, let's use [Bootstrap's Pager](https://react-bootstrap.github.io/components/pagination/)